### ISSUE TITLE
Notebook IntelliSense incorrectly reports code as unreachable

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/pylance-release#2285]
From our test plan,

Open the src/test/datascience/manualTestFiles/manualTestFile.py in VSCode.
Run all of the cells in the file.
Interactive Window should open
Export the cells in the interactive window and open the notebook editor

The cells below the exception cell are incorrectly marked as unreachable. We should probably suppress these types of warnings, they create lots of noise and might contribute to Pylance churn.

![image](https://user-images.githubusercontent.com/30305945/130703745-e9418b0b-0187-45ee-a43a-b5830c1cae4f.png)

**Comment by rchiodo** [comment]
Another repo, have two cells in a notebook like so:

```
raise Exception('fail')
```
```
print('stuff')
```
Second cell will be marked as unreachable

![image](https://user-images.githubusercontent.com/19672699/150883895-b4aaff4d-33c5-4a24-9003-974869c04557.png)

**Comment by luabud** [comment]
@debonte can you repro this with the LSP prototype?

**Comment by debonte** [comment]
@luabud, yes this will be fixed by the LSP notebooks work since each cell will be a treated as a separate document.
